project/ws/app/src/lib/routes/home/routes/marketplace-provider/components/market-place-dashboard/market-place-dashboard.component.scss

```diff
@@ -62,4 +63,22 @@
     position: unset !important;
     margin: 0px !important;
   }
-}
\ No newline at end of file
+}
+
+::ng-deep .container-balanced {
+  padding: 16px 40px !important;
```

Comment:
make sure this wont effect in other components, always use custom parent class along with ng deep